### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by davidhalter**
@blueyed brought up a few small issues that I've fixed here.

### CODE REVIEW

**Comment by davidhalter** [comment]
Thanks for all the reviews @blueyed!

**Comment by davidhalter** [comment]
I'm happy to use pull requests again, but I'm not sure how much time you want to invest into jedi-vim. I stopped using them a while ago, because I thought you had lost interest :).

File: test/vspec/signatures.vim [hunk 1]
```diff
@@ -47,16 +47,13 @@ describe 'signatures'
     it 'highlights correct argument'
         noautocmd normal o
         doautocmd CursorHoldI
-        noautocmd normal iformat(42, 2)
-        " Move to "=" - hightlights "sep=...".
-        noautocmd normal h
+        noautocmd normal iformat(42, "x")
+        " Move to x- hightlights "x".
```
Comment (blueyed) [inline test/vspec/signatures.vim:51]:
```suggestion
        " Move to x - highlights "x".
```

**Review by blueyed — APPROVED** [review]

**Comment by davidhalter** [comment]
You seem to be having the same weird times to be awake :)

**Pull request merged**